presto-main/src/test/java/com/facebook/presto/operator/spiller/BenchmarkBinaryFileSpiller.java

```diff
@@ -142,6 +142,7 @@ public void tearDown()
                     DOUBLE.writeDouble(pageBuilder.getBlockBuilder(4), lineItem.getExtendedPrice());
                 }
                 pages.add(pageBuilder.build());
+                pageBuilder.reset();
```

Comment:
Can you suggest a better title? I am a new contributor to Presto so will help to get your suggestion.